sample-dcrdata.conf

```diff
@@ -74,3 +74,8 @@
 ; comma-separated list. Currently available: coinbase, coindesk, binance,
 ; bittrex, dragonex, huobi, poloniex
 ; disable-exchange=dragonex,huobi
+
+; Pull exchange data from a dcrrates server at the network address given by
+; ratemaster. Requires the server's TLS certificate. 
+;ratemaster=
+;ratecert=
```

Comment:
ratecert=rates.cert and was expecting for it to look in ~/.dcrdata/ for that, but it didn't find it. Seems it was looking in the wording directory, or maybe the path of the binary(?). If that's consistent with how dcrdcert works, that's OK, but I was surprised.